doc/release/1.16.0-notes.rst

```diff
@@ -279,6 +279,13 @@ The ``np.core.records.fromfile`` function now supports ``pathlib.Path``
 and other path-like objects in addition to a file object. Furthermore, the
 ``np.load`` function now also supports path-like objects when
 using memory mapping (``mmap_mode`` keyword argument).
+``ndrange`` a multi-dimensional range-like object
```

Comment:
The newlines got lost here, probably due to rebase. Add a newline above and remove one below the heading.

Reply:
thanks